### PR TITLE
Add take_part format json schema

### PR DESCRIPTION
**Pull request by gpeng**
This is the json schema and example for the take_part format.

[Trello](https://trello.com/c/2A9AMxoL)

### CODE REVIEW

File: formats/take_part/frontend/examples/take_part.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
...d how your own opinion differs from the political parties.\r\n\r\n### Getting officially nominated\r\n\r\nWhether you've been selected by a party or are standing as an independent candidate, you must make sure that you are officially nominated as the election date draws nearer. This means getting 10 people to sign your nomination papers (signatories must be registered electors in the ward where you wish to stand) available from your local council’s democratic services department. \r\n\r\nYou must also give your consent in writing to your nomination. All the necessary documents must be submitted 19 working days before the day of the election.\r\n\r\nFor more detailed information please visit the [Be a councillor website](http://www.beacouncillor.org.uk)\r\nor download the [Be a councillor guide](http://www.local.gov.uk/c/document_library/get_file?uuid=fa4de86d-1009-4b58-a9e7-3103fe3d9a36&groupId=10180).\r\n\r\nThere is [additional support](http://www.gov.uk/government/policies/creating-a-fairer-and-more-equal-society/supporting-pages/making-it-easier-for-disabled-people-to-stand-for-election) available to you if you're disabled and considering standing for elected office. \r\n\r\n",
```
Comment (jamiecobbett) [inline formats/take_part/frontend/examples/take_part.json]: This is Govspeak, but I think it should be HTML.

File: formats/take_part/publisher/links.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  // "required": [
+  //   "organisations"
+  // ],
```
Comment (jamiecobbett) [inline formats/take_part/publisher/links.json]: These commented out lines should be deleted as they are invalid JSON. Is this file needed if there are no custom links?

File: dist/formats/take_part/publisher/schema.json [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+          "enum": [
+            "preserve",
+            "ignore"
+          ],
+          "description": "For prefix redirects, preserve or ignore the rest of the fullpath. For exact, preserve or ignore the querystring."
+        }
+      }
+    }
+  }
+}
```
Comment (benlovell) [inline dist/formats/take_part/publisher/schema.json]: This file (and others) are missing trailing newlines per the POSIX standard.

File: formats/take_part/publisher/details.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "alt_text": {
+          "type": "string"
+        }
+      }
+    },
+    "ordering": {
```
Comment (boffbowsh) [inline formats/take_part/publisher/details.json]: This isn't required for publishing individual Take Part pages, the Get Involved page will still be rendered by Whitehall so will know the order in which to render them.

File: dist/formats/take_part/publisher/schema.json (hunk 3 above)
Comment (jamiecobbett) [inline dist/formats/take_part/publisher/schema.json]: In general I agree, but the compiled schemas don't get a trailing newline, so it is consistent with the others.

Comment (jamiecobbett) [inline dist/formats/take_part/publisher/schema.json]: Although, you're right, we could/should add them to the source files.

File: formats/take_part/frontend/examples/take_part.json (hunk 1 above)
Comment (gpeng) [inline formats/take_part/frontend/examples/take_part.json]: Fixed in commit 14606f8

File: formats/take_part/publisher/links.json (hunk 2 above)
Comment (gpeng) [inline formats/take_part/publisher/links.json]: Fixed in 14606f8

File: formats/take_part/publisher/details.json (hunk 4 above)
Comment (gpeng) [inline formats/take_part/publisher/details.json]: Fixed in eb1bcb9

**Comment by jamiecobbett** [comment]
Needs a rebase (and probably a recompile).

**Comment by boffbowsh** [comment]
:sparkles: :star2: :star: :+1: :star: :star2: :sparkles:

**Comment by jamiecobbett** [comment]
:part_alternation_mark: :fireworks:
